### PR TITLE
Remove functionality duplicates and clean up

### PR DESCRIPTION
**Pull request by vanderhe**
Minor (cosmetic) changes to the Waveplot code-base + duplicate determinant routine removed + a question:
Is the `TSlaterOrbital_assign` subroutine still relevant or an artifact of a formerly required workaround that may be removed now?

### CODE REVIEW

File: app/waveplot/waveplot.F90 [hunk 1]
```diff
@@ -75,7 +75,7 @@ program waveplot
   !> MPI environment
   type(TMpiEnv) :: mpiEnv
 
-  ! As this is serial code, trap for run time execution on more than 1 processor with an mpi enabled
+  ! As this is serial code, trap for run time execution on more than 1 processor with an MPI enabled
```
Comment (bhourahine) [inline app/waveplot/waveplot.F90:78]:
```suggestion
  ! As this is serial code, trap for run time execution on more than 1 processor with MPI enabled
```

Comment (vanderhe) [reply app/waveplot/waveplot.F90]: Applied, thanks.

File: app/waveplot/slater.F90 [hunk 2]
```diff
@@ -7,55 +7,66 @@
...
+  public :: realTessY
+  public :: TSlaterOrbital, TSlaterOrbital_init, getValue, assignment(=)
 
-  !> Data for STOs
+
+  !> Data type for STOs.
   type TSlaterOrbital
     private
+
+    !>
```
Comment (bhourahine) [inline app/waveplot/slater.F90:26]: Most of this type's component comments match the variables in the TSlaterOrbital_getValue_explicit subroutine, so could be copied over. Those comments aren't completely clear, but are better than nothing. Then gridValue is the radial grid and gridDist the spacing and nGrid the number of points (there is reduncancy between these quantities, but it's how it is coded).

Comment (vanderhe) [reply app/waveplot/slater.F90]: You're right, I've copied over/added the missing comments.

File: app/waveplot/initwaveplot.F90 [hunk 3]
```diff
@@ -376,7 +377,7 @@ subroutine TProgramVariables_init(this)
   end subroutine TProgramVariables_init
 
 
-  !> Interpret the information stored in detailed.xml
+  !> Interprets the information stored in detailed.xml.
```
Comment (bhourahine) [inline app/waveplot/initwaveplot.F90:380]: Original comment was gramatically [OK](https://www.merriam-webster.com/dictionary/interpret).

File: app/waveplot/initwaveplot.F90 [hunk 4]
```diff
@@ -488,7 +489,7 @@ subroutine readGeometry(geo, geonode)
   end subroutine readGeometry
 
 
-  !> Interpret the options.
+  !> Interprets the options.
```
Comment (bhourahine) [inline app/waveplot/initwaveplot.F90:492]: ditto

File: app/waveplot/initwaveplot.F90 [hunk 5]
```diff
@@ -450,7 +451,7 @@ subroutine readDetailed(this, detailed, tGroundState, kPointsWeights)
   end subroutine readDetailed
 
 
-  !> Read in the geometry stored as xml in internal or gen format.
+  !> Reads in the geometry stored as .xml in internal or .gen format.
```
Comment (bhourahine) [inline app/waveplot/initwaveplot.F90:454]: ditto

File: app/waveplot/initwaveplot.F90 [hunk 6]
```diff
@@ -775,7 +776,7 @@ subroutine readBasis(this, node, speciesNames)
   end subroutine readBasis
 
 
-  !> Read in basis function for a species.
+  !> Reads in basis function for a species.
```
Comment (bhourahine) [inline app/waveplot/initwaveplot.F90:779]: ditto

**Review by bhourahine — APPROVED** [review]
Please rebase before merging.

**Comment by vanderhe** [comment]
> Please rebase before merging.

All done, thanks.

**Pull request merged**